desktop/install/fedora.md

```diff
@@ -22,9 +22,9 @@ Additionally, for a Gnome Desktop environment you must install AppIndicator and
 
 For non-Gnome Desktop environments, `gnome-terminal` must be installed:
 
-```console
-$ sudo dnf install gnome-terminal
-```
+   ```console
+   $ sudo dnf install gnome-terminal
+   ```
```

Comment:
◀️ Not a step so needs deindent

````suggestion
```console
$ sudo dnf install gnome-terminal
```
````